src/main/java/kotlinx/reflect/lite/builtins/JavaToKotlinClassMap.kt

```diff
@@ -0,0 +1,150 @@
+package kotlinx.reflect.lite.builtins
```

Comment:
Also, this file copies a lot of logic from https://github.com/JetBrains/kotlin/blob/master/core/compiler.common.jvm/src/org/jetbrains/kotlin/builtins/jvm/JavaToKotlinClassMap.kt. I think it would be better to add links to the places where the code was taken from. This way, there would be a clear way to see how the code evolved, starting from the moment it was written. Also, if our code stops working with some new version of the compiler, we will know the specific places where a desync could happen.

Reply:
Yes, that will be useful 🙂 I'll add the references to the code that I reuse from the compiler. And I guess we will be able to get rid of the most part of it.